### PR TITLE
News: GRASS GIS 8.4.0 released

### PR DESCRIPTION
**Pull request by neteler**
News item for the GRASS GIS 8.4.0 release.

~~:mega: Please someone donate a screenshot! :mega:~~

Update of the relevant files:

- [x] data/grass.json
- [x] content/about/history/releases.md
- [x] content/news/2024_07_27_GRASS_GIS_8_4_0_released.md
- [x] DOI: [10.5281/zenodo.13102854](https://doi.org/10.5281/zenodo.13102854) added
- [x] static/images/news/grass840_TODO_TODO.png

image

### CODE REVIEW

File: content/news/2024_07_27_GRASS_GIS_8_4_0_released.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+We'd like to **thank the translators of all languages** for their
+ongoing support!
+
+### Full list of changes and contributors
+
+For all 520 changes, see our detailed announcement with the full list of
+features and bugs fixed at
+[GitHub / Releases / 8.4.0](https://github.com/OSGeo/grass/releases/tag/8.4.0).
+
+Thanks to all contributors!
```
Comment (veroandreo) [inline content/news/2024_07_27_GRASS_GIS_8_4_0_released.md:42]:
```suggestion
Thanks to all contributors and financial supporters!
```

File: content/news/2024_07_27_GRASS_GIS_8_4_0_released.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+**About GRASS GIS**
+
+The Geographic Resources Analysis Support System
+([https://grass.osgeo.org/](/)), commonly referred to as GRASS GIS, is
+an Open Source Geographic Information System providing powerful raster,
+vector and [geospatial processing capabilities](https://grass.osgeo.org/learn/overview/).
+It can be used either as a stand-alone application, as backend for other
+software packages such as QGIS and R, or in the cloud. It is
+distributed freely under the terms of the GNU General Public License (GPL).
+GRASS GIS is a founding member of the Open Source Geospatial Foundation (OSGeo).
```
Comment (veroandreo) [inline content/news/2024_07_27_GRASS_GIS_8_4_0_released.md:87]: Can we remove this chunk, please? :pray:

**Review by veroandreo — COMMENTED** [review]
Thanks @neteler! Looks good, I'd only remove the last paragraph *About GRASS GIS*. I'll provide a screenshot in a moment.

**Comment by veroandreo** [comment]
Screenshot added:

image

File: content/news/2024_07_27_GRASS_GIS_8_4_0_released.md (hunk 2 above)
Comment (neteler) [reply content/news/2024_07_27_GRASS_GIS_8_4_0_released.md:87]: Why should it go? It is pretty standard for press releases

Comment (veroandreo) [reply content/news/2024_07_27_GRASS_GIS_8_4_0_released.md:87]: Well, the news is on our website and the press release will point to it, right? So, I think a self description is not needed there. I don't see other projects having such a paragraph in their release notes either. 

What about replacing it with a *More about GRASS GIS* section and a bullet point list of links? Something like:

```
#### More about GRASS GIS

- See our list of [tutorials](https://grass.osgeo.org/learn/tutorials/).
- See GRASS [tools manual pages](https://grass.osgeo.org/grass-stable/manuals/index.html).
- [Get in touch with us](https://grass.osgeo.org/support/community/).
- Feel like supporting the project? [Donate](https://opencollective.com/osgeo/projects/grass/contribute).
```

Comment (neteler) [reply content/news/2024_07_27_GRASS_GIS_8_4_0_released.md:87]: Traditionally, the press release was sent to multiple portals etc. As this no longer really happens (my lack of energy in this regard) I have reconsidered your suggestion and the new list of items.

Comment (veroandreo) [reply content/news/2024_07_27_GRASS_GIS_8_4_0_released.md]: Thanks! I replaced it now

image

Comment (neteler) [reply content/news/2024_07_27_GRASS_GIS_8_4_0_released.md]: Thanks a lot!

**Review by veroandreo — APPROVED** [review]

**Pull request merged**